Source/WebCore/platform/sql/SQLiteDatabase.cpp

```diff
@@ -155,7 +159,16 @@ bool SQLiteDatabase::open(const String& filename, OpenMode openMode)
         int result = SQLITE_OK;
         {
             SQLiteTransactionInProgressAutoCounter transactionCounter;
-            result = sqlite3_open_v2(FileSystem::fileSystemRepresentation(filename).data(), &m_db, flags, nullptr);
+            auto fsFilename = FileSystem::fileSystemRepresentation(filename);
+            result = sqlite3_open_v2(fsFilename.data(), &m_db, flags, nullptr);
+#if PLATFORM(COCOA)
+            if (result == SQLITE_OK && options.contains(OpenOptions::CanSuspendWhileLocked)) {
+                char excluded = 0xff;
+                auto err = setxattr(fsFilename.data(), "com.apple.runningboard.can-suspend-locked", &excluded, sizeof(excluded), 0, 0);
```

Comment:
I'm surprised `com.apple.runningboard.can-suspend-locked` isn't a constant defined somewhere.

Reply:
`setxattr` is already permitted in the Network Process sandbox, so I think this change should be fine. I don't believe this code is accessed outside of the Network Process.

Reply:
It doesn't seem com.apple.runningboard.can-suspend-locked is defined in a constant anywhere.